src/components/Dropdown/Dropdown.module.scss

```
@@ -0,0 +1,59 @@
+.initial {
+  width: 13.5625rem;
+  height: 3rem;
+  border-radius: 0.375rem;
+  border: 0.0625rem solid $color-gray-d9d9d9;
+  padding: 0.61rem 1rem;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: $color-gray-9fa6b2;
+  font-size: 1rem;
+
+  @include mobile {
+    width: 100%;
+  }
+}
+
+.active {
+  border: 0.0625rem solid $color-violet-5534da;
+}
+
+.dropdown {
+  position: absolute;
+  width: 13.5625rem;
+  height: auto;
+  left: -0.1rem;
+  top: 3rem;
+  box-shadow: 0 0.25rem 1.25rem 0 rgba(0, 0, 0, 0.08);
+  padding: 0.6rem 0;
+  border: 0.0625rem solid $color-gray-d9d9d9;
+  background-color: $color-white-ffffff;
+  border-radius: 0.375rem;
+  z-index: 999;
+
+  @include mobile {
+    width: 100%;
+  }
+}
+
+.list {
+  padding: 0.6rem 0 0.6rem 2.25rem;
+  position: relative;
+}
+
+.list:hover {
+  background-color: $color-gray-fafafa;
+}
```

p4; 이렇게 변경 가능할 거 같아요!
```suggestion
.list {
  padding: 0.6rem 0 0.6rem 2.25rem;
  position: relative;
  
  &:hover {
  background-color: $color-gray-fafafa;
}
}
```